soapui/src/main/java/com/eviware/soapui/tools/SplashScreenStomper.java

```diff
@@ -45,6 +46,7 @@ public static void main(String arg[]) throws IOException {
         }
 
         ImageIO.write(bufferedImage, "png", new File(outputFilePath));
+        ImageIO.write(bufferedImage, "png", new File(outputFilePathBat));
```

Comment:
instead of this, I guess it makes more sense to handle 2x image by this code (it will require position changes). I'm ok with having only low resolution splash bat running.